pym/bob/input.py

```diff
@@ -3431,8 +3464,19 @@ def __parse(self, envOverrides, platform, recipesRoot=""):
             self.__parseUserConfig(os.path.join(os.environ.get('XDG_CONFIG_HOME',
                 os.path.join(os.path.expanduser("~"), '.config')), 'bob', 'default.yaml'))
 
+        osEnv = Env(os.environ)
+        osEnv.setFuns(self.__stringFunctions)
+        env = Env({ k : osEnv.substitute(v, k) for (k, v) in
+            self.__defaultEnv.items() })
+        env.setFuns(self.__stringFunctions)
+        env.update(envOverrides)
+        env["BOB_HOST_PLATFORM"] = platform
+        self.__rootEnv = env
+
+        if noLayers:
```

Comment:
:thinking: you are right. OTOH this would only effect policies set in the recipes-root not in any sublayer since the layers are not parsed again after a checkout of a new layer...

Reply:
Luckily, this is not an issue. The policies are determined by the root layer. It's only checked that sub-layers do not contradict them.